front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -32,6 +34,30 @@ const SimulationReportSheet = ({
     path_number2: 'n°YYYYYY',
   };
 
+  const [mapImageUrl, setMapImageUrl] = useState<string | null>(null);
+
+  // Convert image to JPEG
+  useEffect(() => {
+    if (mapCanvas) {
+      base64ToJpeg(mapCanvas, 0.8).then((blob) => {
+        if (blob) {
```

Comment:
Alright, one last nit: I think this `if` is unnecessary now that `blob` cannot be `null`.

Reply:
fixed, ty for your review :)